Custom shaders no longer possible

Custom shader support was added in #265. However, since then, the TexturedColored shader has been created bbut not externalized, so now it's no longer possible to construct GlGraphics with custom shaders.

```error[E0433]: failed to resolve: use of undeclared type TexturedColor
   --> src/main.rs:202:76
    |
202 |         gl: GlGraphics::from_pieces(Colored::new(glsl), test_shader(glsl), TexturedColor::new(glsl)),
    |                                                                            ^^^^^^^^^^^^^ use of undeclared type TexturedColor```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,7 +1,7 @@
 [package]
 
 name = "piston2d-opengl_graphics"
-version = "0.88.0"
+version = "0.89.0"
 edition = "2018"
 authors = [
     "bvssvni <bvssvni@gmail.com>",

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -14,7 +14,7 @@ extern crate shaders_graphics2d as shaders;
 extern crate texture as texture_lib;
 extern crate viewport;
 
-pub use crate::back_end::{Colored, GlGraphics, Textured};
+pub use crate::back_end::{Colored, GlGraphics, Textured, TexturedColor};
 pub use crate::texture::Texture;
 pub use shader_version::glsl::GLSL;
 pub use shader_version::{OpenGL, Shaders};

